composer.json

```diff
@@ -41,8 +41,8 @@
         "phpstan": "phpstan analyse src --level=max -c phpstan.neon --no-progress --ansi",
         "sniffer:check": "phpcs --standard=phpcs.xml",
         "sniffer:fix": "phpcbf --standard=phpcs.xml",
-        "test": "phpunit --configuration phpunit.xml --do-not-cache-result --colors=always",
-        "test:coverage": "phpunit --configuration phpunit.xml --do-not-cache-result --colors=always --coverage-clover build/logs/clover.xml --coverage-html build/coverage"
+        "test": "phpunit --configuration phpunit.xml --do-not-cache-result",
```

Comment:
Why did you removed the `--colors=always` option?

Reply:
> The colors attribute is defined on phpunit.xml and it's unnecessary to add --colors=always option during PHPUnit tests running.

This above explanation is mentioned on my PR description :).